docs/alerts/monitors/create-monitor.md

```diff
@@ -250,6 +233,23 @@ Log monitor triggers are evaluated by balancing the requirement of timely alert
 
 <img src={useBaseUrl('img/monitors/metrics-query.png')} alt="metrics query.png" style={{border: '1px solid gray'}} width="600" />
 
+#### Prerequisites
+
+To fully leverage AI-driven alerts for metrics monitors, you'll need:
+
+* **Automation Service**. Required for linking playbooks to metrics-based monitors.
+* **Metrics data**. Metrics data must be sent to Sumo Logic for anomaly detection.
+* **Metrics aggregation**. Queries that return multiple time series should be aggregated (for example, using `sum` or `avg` operators) before applying anomaly detection.
+
+Examples:
+
+* **Cluster anomalies detection**. A user configures alerts for anomalies when 5 out of 10 data points in a 10-minute window exceed the baseline, allowing for precision in spiky environments.
+* **Automating resolution with playbooks**. A playbook responds to CPU usage anomalies by gathering logs, notifying teams, and restarting affected servers.
+
+:::info Limitations
+Anomaly detection applies to one time series at a time. Multi-time series queries must be aggregated before detection.
```

Comment:
We'll require aggregation to be applied at the end of all metric anomaly monitors trigger queries

Reply:
thank you! changed to "Anomaly detection applies to one time series at a time. All metrics anomaly monitor trigger queries must have aggregation applied at the end of the query before detection."